pyproject.toml

```diff
@@ -40,3 +40,20 @@ select = [
     # Pyupgrade
     "UP",
 ]
+
+[tool.cibuildwheel]
+build-verbosity = 1
+skip = [
+    "cp38-*",
+    "cp39-*",
+    "cp314t-*",
```

Comment:
given that both 3.13 and 3.14 wheels were missing I assumed that free-threading wasn't supported yet. Note that cibuildwheel 3.2 and newer just enable it by default for 3.14, not 3.13 (because it was marked as experimental for that version), so, to be conservative, and because it's potentially much more work to add, I just disabled this one explicitly.